### PR TITLE
163 Add pagination for all admin resources

### LINKED ISSUE #163: Add pagination for all admin resources
- kaminari
- default per_page 25

### PR DESCRIPTION
**Pull request by denys281**
Fixed #163
(note: this line will auto close issue)

Generated kaminari config
Generated kaminari views (semantic-ui)
Fixed minor style issues
Added more seeds for Events, Venues, Emails, Friends, Goals, Talks

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -30,6 +30,41 @@
   map_url: 'https://www.google.com/maps/place/%D0%B2%D1%83%D0%BB%D0%B8%D1%86%D1%8F+%D0%97%D0%B0%D0%BC%D0%BA%D0%BD%D0%B5%D0%BD%D0%B0,+9,+%D0%9B%D1%8C%D0%B2%D1%96%D0%B2,+%D0%9B%D1%8C%D0%B2%D1%96%D0%B2%D1%81%D1%8C%D0%BA%D0%B0+%D0%BE%D0%B1%D0%BB%D0%B0%D1%81%D1%82%D1%8C,+%D0%A3%D0%BA%D1%80%D0%B0%D1%97%D0%BD%D0%B0/@49.8404509,24.0134988,17z/data=!3m1!4b1!4m5!3m4!1s0x473add76e1c6ee3b:0xda08ff9150b3da68!8m2!3d49.8404475!4d24.0156875'
 ).first_or_create!
 
+100.times do
```
Comment (sveredyuk) [inline db/seeds.rb]: Please, add check for `Rails.env.development?` because we don't need so much dummy data in production app.

**Review by sveredyuk — APPROVED** [review]

**Review by sveredyuk — APPROVED** [review]
Well done

**Comment by sveredyuk** [comment]
@denys281 Squash all commits into 1 with nice message and force push. I will merge.

**Comment by denys281** [comment]
@sveredyuk squashed.

**Pull request merged**